### PR TITLE
ci: run lscpu at end of build

### PR DESCRIPTION
**Pull request by nebril**
We have been running into cpu frequency issues in our ci, this change
will allow us to check whether the build was affected by the problem.

### CODE REVIEW

**Comment by nebril** [comment]
test-focus K8sDemos

**Comment by nebril** [comment]
test-runtime

File: jenkinsfiles/ginkgo-kernel.Jenkinsfile [hunk 1]
```diff
@@ -193,6 +193,7 @@ pipeline {
 
     post {
         always {
+            sh 'lscpu'
```
Comment (pchaigno) [inline jenkinsfiles/ginkgo-kernel.Jenkinsfile:196]: Don't we need to the same for `ginkgo-kubernetes-all.Jenkinsfile` and `ginkgo.Jenkinsfile`?

**Comment by nebril** [comment]
test-focus K8sDemos

**Review by pchaigno — APPROVED** [review]

File: jenkinsfiles/Jenkinsfile.nightly [hunk 2]
```diff
@@ -133,6 +133,7 @@ pipeline {
     }
     post {
         always {
+            sh 'lscpu'
```
Comment (joestringer) [inline jenkinsfiles/Jenkinsfile.nightly:136]: I note that some lines use double-quote and others use single-quote, but seems like both are respected.

**Review by joestringer — APPROVED** [review]

**Pull request merged**